modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/FieldFeedback.tsx

```diff
@@ -15,30 +15,25 @@ export function FieldFeedback({
 	warningMessage,
 	...otherProps
 }: IProps) {
-	if (!errorMessage && !helpMessage && !warningMessage) {
-		return null;
-	}
+	const message =
+		warningMessage && !errorMessage ? warningMessage : errorMessage;
+
+	const symbol =
+		warningMessage && !errorMessage ? 'warning-full' : 'exclamation-full';
```

Comment:
In these two ternary statements you are doing the exact same check. In this situation it would be better to only make the check once and reuse it. This can be done with an `if` block or by declaring a variable and using that variable in the ternary statements: `const showWarning = warningMessage && !errorMessage;`

Reply:
Fixed!